### PR TITLE
Fix non-idiomatic loop

### PR DESCRIPTION
**Pull request by jianglingxia**
discard the first item in the range fix the format

### CODE REVIEW

**Comment by feiskyer** [comment]
@jianglingxia It is wrong to use range variables for pointers (should make a copy instead). Have you validated whether the patch is actually working?

**Comment by thockin** [comment]
This is a trivial change that doesn't seem to match the comments..?

**Comment by feiskyer** [comment]
sorry, I'm wrong on this.

@k8s-bot ok to test

**Comment by thockin** [comment]
I still don't understand what the PR title means - it has no relation to the code.

**Comment by feiskyer** [comment]
@thockin me too 😓

**Comment by jianglingxia** [comment]
just fix the format ,thanks

**Comment by thockin** [comment]
@k8s-bot test this

**Comment by jianglingxia** [comment]
why the jenkins job failed ? can I repull the commit once again?

**Comment by thockin** [comment]
@k8s-bot test this

File: pkg/client/listers/apps/internalversion/statefulset_expansion.go [hunk 1]
```diff
@@ -52,8 +52,7 @@ func (s *statefulSetLister) GetPodStatefulSets(pod *api.Pod) ([]*apps.StatefulSe
 	}
 
 	var psList []*apps.StatefulSet
-	for i := range list {
-		ps = list[i]
+	for _, ps := range list {
```
Comment (wojtek-t) [inline pkg/client/listers/apps/internalversion/statefulset_expansion.go]: You need to remove "ps" definition above.

**Comment by wojtek-t** [comment]
@jianglingxia - moving out of 1.7 milestone, since it doesn't compile and there was no update since I asked for fix 6 days ago.

**Comment by kargakis** [comment]
I would be cautious with switching these to use the range value:
https://tam7t.com/golang-range-and-pointers/
We have had this problem before in other parts of k8s.

**Comment by thockin** [comment]
ping

**Pull request closed without merging**